config.docker.yaml

```diff
@@ -0,0 +1,46 @@
+issuer: {{ getenv "DEX_ISSUER" "http://127.0.0.1:5556/dex" }}
+
```

Comment:
Maybe add a comment that this config file is only an example / for development purposes?

And maybe mention in another comment that for escaping "unfriendly" input like passwords, this function could be used for escaping? https://docs.gomplate.ca/functions/strings/#strings-squote (Sorry haven't tested it yet, but I'm assuming that such escaping is not done automatically by Gomplate)

Reply:
I think that makes sense. Let's add a comment